lib/src/main/java/com/diffplug/spotless/java/GoogleJavaFormatStep.java

```diff
@@ -76,16 +76,26 @@ public static FormatterStep create(String version, String style, Provisioner pro
 
 	/** Creates a step which formats everything - code, import order, and unused imports - and optionally reflows long strings. */
 	public static FormatterStep create(String version, String style, Provisioner provisioner, boolean reflowLongStrings) {
+		return create(MAVEN_COORDINATE, version, style, provisioner, reflowLongStrings);
+	}
+
+	/** Creates a step which formats everything - groupArtifact, code, import order, and unused imports - and optionally reflows long strings. */
+	public static FormatterStep create(String groupArtifact, String version, String style, Provisioner provisioner, boolean reflowLongStrings) {
+		Objects.requireNonNull(groupArtifact, "artifact");
```

Comment:
Is it expected `group:artifact` or `group:artifact:`? Should do some error-checking here with clear messages - must have one `:`, no more no less.